modules/storage-persistent-storage-pvc-volumeattributesclass.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-persistent-storage.adoc
+//
+
+:_mod-docs-content-type: CONCEPT 
+[id="storage-persistent-storage-pvc-volumeattributesclass_{context}"]
+= Volume Attributes Classes
+
+Volume Attributes Classes provide a way for administrators to describe "classes" of storage they offer. Different classes might correspond to different quality-of-service levels.
+
+:FeatureName: Volume Attributes Classes
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+Volume Attributes Classes in {product-title} is available only with AWS Elastic Block Storage (EBS) and Google Cloud Platform (GCP) persistent disk (PD) Container Storage Interface (CSI).
+
+You can apply a Volume Attributes Class to a persistent volume claim (PVC). If a new Volume Attributes Class becomes available in the cluster, a user can update the PVC with the new Volume Attributes Class if needed.
+
+Volume Attributes Classes have parameters that describe volumes belonging to them. Different parameters can be accepted depending on the provisioner or the resizer. If a parameter is omitted, the default is used at volume provisioning. If you apply the PVC with a different Volume Attributes Class with omitted parameters, the default value of the parameters might be used depending on the CSI driver implementation. For more information, see the related CSI driver documentation.
```

Comment:
> Different parameters can be accepted depending on the provisioner or the resizer. 

No the parameters are property of CSI driver. And are opaque to provisioner or resizer.

Reply:
I just deleted that sentence.